shared/oval/accounts_max_concurrent_login_sessions.xml

```diff
@@ -54,7 +54,7 @@
     <ind:path>/etc/security/limits.d</ind:path>
     <ind:filename operation="pattern match">^.*\.conf$</ind:filename>
     <ind:pattern operation="pattern match">^[\s]*\*[\s]+(?:(?:hard)|(?:-))[\s]+maxlogins</ind:pattern>
-    <ind:instance datatype="int">1</ind:instance>
+    <ind:instance datatype="int" operation="greater than or equal">0</ind:instance>
```

Comment:
I am probably missing something. Could you please explain why you used 0 here and not 1?